projects/app/src/app/features/advanced-search-playground/advanced-search-playground.component.html

```diff
@@ -0,0 +1,49 @@
+<lib-view-layout title="Advanced Search Playground"
+                 [busy]="!items"
+                 [actions]="actions"
+                 [meta-items]="metaItems">
+    <lib-list-view [items]="items"
+                   [show-pager]="true"
+                   [search-options]="searchOptions"
+                   [pager-position]="'both'"
+                   [page]="page"
+                   [filters]="filters"
```

Comment:
To filters το παίρνει από το BaseListComponent που κάνεις inherit. Μπορείς να το γράψεις specifically μόνο αν θες να κάνεις overwrite το filters που σου δίνει έτοιμο, αλλιώς δεν χρειάζεται να το δηλώσεις καν
```suggestion

```